queries/ecma/indents.scm

```diff
@@ -59,10 +59,11 @@
 (statement_block
   "{" @indent.branch)
 
-(parenthesized_expression
+((parenthesized_expression
   ("("
     (_)
-    ")" @indent.end))
+    ")" @indent.end)) @_outer
+  (#not-has-parent? @_outer if_statement))
```

Comment:
Can you also remove the unnecessary grouping here

```diff
((parenthesized_expression
-  ("("
-    (_)
-    ")" @indent.end)) @_outer
+  "("
+  (_)
+  ")" @indent.end) @_outer
  (#not-has-parent? @_outer if_statement))
`

Reply:
Is this a pattern that the formatter should handle? or could there be cases where it can't tell if the grouping is redundant? don't know if it's different for non-top-level groupings

Reply:
It's kinda hard to detect useless grouping. Because it may be needed. I think it can be detected with

```
(named_node
  (grouping
    "(" @format.remove
    ")" @format.remove
    .)
```

I have tried to remove with this before, but because this cannot reliably detect quantifiers. May have been fixed since.

Reply:
Ah, makes sense